src/AmqpBundle/Resources/config/sandbox_services.yml

```diff
@@ -0,0 +1,6 @@
+parameters:
+    m6_web_amqp.exchange.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopExchange'
+    m6_web_amqp.queue.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopQueue'
+    m6_web_amqp.connection.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopConnection'
+    m6_web_amqp.channel.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopChannel'
```

Comment:
I understand your POV 👌 

Moving mock classes seems the best option to me... but it means breaking changes (even if ridiculously small)... so I'm not sure

I think we should ask for someone else opinion (maybe @fabdsp?)

Besides, if you move mock classes, or if we keep the `NoopStuff`, I think these classes should be renamed `NullStuff` (i.e. `NullAMQPChannel`) to be more "explicit" or "standard"